### DISCUSSION TITLE
Quoting messages for reply in a group causes issues for IOS users

### DISCUSSION THREAD

**Discussion by deven96** [tulir/whatsmeow#148] [WhatsApp protocol Q&A]
I'm not certain if it's an IOS issue but so far it seems to be. Quoting a message using `ContextInfo` where it looks something like

```go
&waProto.ContextInfo{
		StanzaId:      &event.Info.ID,
		Participant:   proto.String(event.Info.Sender.String()),
		QuotedMessage: event.Message,
	}
```
Seems to quote the original message in a group chat just fine from an android perspective. However it's made to appear to IOS users like they sent the original quoted message 
![20220519_205613](https://user-images.githubusercontent.com/23453888/169393790-74b1001d-964d-4bed-9942-d4e92e5736f9.jpg)

The original message shows up for IOS User as being sent by "You". While below is an example for android user
![IMG-20220519-WA0120](https://user-images.githubusercontent.com/23453888/169394201-f9043f31-de33-469d-bf5b-ae1d09acd258.jpg)

**Comment by rsaosk** [comment]
Similar problem. How to solve?

**Comment by hajsf** [comment]
@deven96  / @rsaosk  can you share your full code for sending the quote message

**Reply by rsaosk** [reply]
```
_, _ = client.SendMessage(context.Background(), event.Info.Chat, "", &waProto.Message{
				ExtendedTextMessage: &waProto.ExtendedTextMessage{
					Text: proto.String("Hello, World!"),
					ContextInfo: &waProto.ContextInfo{
						StanzaId:      proto.String(event.Info.ID),
						Participant:   proto.String(event.Info.Sender.String()),
						QuotedMessage: event.Message,
					},
				},
			})
```

**Reply by solar-meadow** [reply]
Thank you bro. you solve my problem

**Comment by rsaosk** [comment]
The problem is still relevant(

**Reply by krismandev** [reply]
can the Message object only be obtained through event.Message ?

**Comment by ichsansaid** [comment]
i still have the same problem
